File: Submissions/T15/M2.py
n = int(input("Masukkan N : "))

a = 0
b = 1

print("Deret :", end=" ")

for i in range(1, n + 1):
    print(a, end=" ")

    if i == n:
        hasil = a

    temp = a + b
    a = b
    b = temp

print()
print(f"Fibonacci ke-{n} : {hasil}")

File: Submissions/T15/M2.cpp
#include <stdio.h>

int main() {
    int n;
    int a = 0, b = 1, temp, hasil;

    printf("Masukkan N : ");
    scanf("%d", &n);

    printf("Deret : ");

    for(int i = 1; i <= n; i++) {
        printf("%d ", a);

        if(i == n)
            hasil = a;

        temp = a + b;
        a = b;
        b = temp;
    }

    printf("\nFibonacci ke-%d : %d\n", n, hasil);

    return 0;
}
